zproject/default_settings.py

```diff
@@ -488,7 +488,7 @@
 STAGING = False
 
 # How long to wait before presence should treat a user as offline.
-OFFLINE_THRESHOLD_SECS = 140
+OFFLINE_THRESHOLD_SECS = 200
 
 # How often a client should ping by asking for presence data of all users.
 PRESENCE_PING_INTERVAL_SECS = 50
```

Comment:
Per https://chat.zulip.org/#narrow/stream/3-backend/topic/UserPresence.20system.20rewrite/near/1506229 I'm wondering if the ping interval should be `60` like in the mobile app or we wanna leave it at `50`. If we leave it at `50`, should `OFFLINE_THRESHOLD_SECS` be `200` indeed, or `170-180` per the `3 x ACTIVE_PING_INTERVAL_MS + 20` rule of thumb mentioned in https://chat.zulip.org/#narrow/stream/3-backend/topic/UserPresence.20system.20rewrite/near/1506197